src/lib/Scenes/Home/Components/Sales/Components/LotsByFollowedArtists.tsx

```diff
@@ -5,34 +5,25 @@ import { get, once } from "lodash"
 import PropTypes from "prop-types"
 import React, { Component } from "react"
 import { FlatList, ScrollView, Text, View } from "react-native"
-import { RelayPaginationProp } from "react-relay"
+import { createPaginationContainer, graphql, RelayPaginationProp } from "react-relay"
 import styled from "styled-components/native"
 import { SectionHeader } from "./SectionHeader"
 
 const DEFAULT_TITLE = "Lots by Artists You Follow"
-export const PAGE_SIZE = 10
-
-const Container = styled.View`padding: 10px;`
-
-interface Props {
-  relay?: RelayPaginationProp
-  title?: string
-  saleArtworks?: {
-    edges: Array<{
-      node: object | null
-    }> | null
-  }
-}
+const PAGE_SIZE = 10
 
 export function LotsByFollowedArtists(props: Props) {
-  const { relay, saleArtworks, title = DEFAULT_TITLE } = props
-
-  const artworks = get(saleArtworks, "edges", []).filter(({ node }) => node.is_biddable).map(({ node }) => node.artwork)
+  const { relay, viewer, title = DEFAULT_TITLE } = props
+  const artworks = get(viewer, "sale_artworks.edges", [])
```

Comment:
Your use of `get` is losing type information and shouldn’t be necessary anyways.